test/ReactViews/Map/Panels/SharePanel/BuildShareLinkSpec.ts

```diff
@@ -55,6 +38,19 @@ const flattenInitSources = (initSources: InitSourceData[]): InitSourceData =>
   );
 
 describe("BuildShareLink", function () {
+  let terria: Terria;
+  let viewState: ViewState;
+
+  beforeEach(function () {
```

Comment:
Moved inside the only `describe()` in this file to avoid https://github.com/joeljeske/karma-parallel/issues/64.